### ISSUE TITLE
WHLs for cuda 11.7, 11.8, and 12.0 for future Releases

### ISSUE THREAD

**Issue by Qubitium** [microsoft/BitBLAS#62]
Currently the bitblas whl support is too limited >= 12.1.  I understand that building so many whl/python/torch combos is a headache but I think it may be worth it.

Include support for all Cuda supported by Torch >= 2.0.0 which need to add 11.7, 11.8, and 12.0 to the WHL builds. 

Reasons:

1. Lots of gpu-poor academics are locked to institution provided envs where drivers are often locked to cuda 11.7, 11.8 version. 
2. Compiling bitblas has large os lib dependency and a simple git clone + build is not possible even on ubuntu without adding ubuntu pkgs. But if env is not ubuntu the problem becomes quite a problem for users that has no clue about builds.
3. Allow 3rd party to fully embed Bitblas without raising Torch/Cuda requirements. GPTQModel, for example, has integrated bitblas right now as a non-optional integration but need to add cuda checks for pkg compat and redirect users to src compile during runtime.

```[tasklist]
### Tasks
- [x] Append cuda 11.7, 11.8, and 12.0 for future Releases
```

**Comment by LeiWang1999** [comment]
Hi @Qubitium , thank you for your attention. Indeed, bitblas is not officially released yet. We are currently working on performance-related optimizations, And there are still many items on our roadmap, such as CI/CD integration and support for VLLM. We are committed to completing these tasks and releasing more WHL packages in our official release.

**Comment by tngh5004** [comment]
We're waiting for this action. Thank you for your efforts :)

**Comment by LeiWang1999** [comment]
@tngh5004 , thanks for you attention, we will arrange this item as soon as possible.

**Comment by LeiWang1999** [comment]
Thanks @tzj-fxz for the fix, we just released 0.0.1.dev13 on pypi, the dependency for cuda 12 has been removed, please feel free to test with:

```bash
pip install bitblas==0.0.1.dev13
```

@Qubitium , @tngh5004

**Comment by tngh5004** [comment]
> Thanks @tzj-fxz for the fix, we just released 0.0.1.dev13 on pypi, the dependency for cuda 12 has been removed, please feel free to test with:
> 
> ```shell
> pip install bitblas==0.0.1.dev13
> ```
> 
> @Qubitium , @tngh5004

Thank you very much, I'm currently busy finalizing my thesis, but I'll experiment with it as soon as possible to see if it works.
